saleor/settings.py

```diff
@@ -269,6 +270,18 @@ def get_bool_from_env(name, default_value):
 
 ENABLE_DEBUG_TOOLBAR = get_bool_from_env("ENABLE_DEBUG_TOOLBAR", False)
 if ENABLE_DEBUG_TOOLBAR:
+    # Ensure the debug toolbar is actually installed
+    try:
+        __import__("debug_toolbar")
+    except ImportError as exc:
+        print(
```

Comment:
What if we only issued a warning and continued to run?

Reply:
Sounds fine to me 👍